homeassistant/components/remember_the_milk.py

```diff
@@ -0,0 +1,249 @@
+"""Component to interact with Remember The Milk.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/remember_the_milk/
+
+Minimum viable product, it currently only support creating new tasks in your
+Remember The Milk (https://www.rememberthemilk.com/) account.
+
+This product uses the Remember The Milk API but is not endorsed or certified
+by Remember The Milk.
+"""
+import logging
+import os
+import json
+import voluptuous as vol
+
+from homeassistant.config import load_yaml_config_file
+from homeassistant.const import (CONF_API_KEY, STATE_OK, CONF_TOKEN, CONF_NAME)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.entity_component import EntityComponent
+
+REQUIREMENTS = ['RtmAPI==0.7.0', 'httplib2==0.10.3']
```

Comment:
Why would you depend on `httplib2` ?

Reply:
You're not using it at all, please remove it.

Reply:
It's a transitive dependency from RtmAPI. 

Even though it's defined in their [setup.py](https://bitbucket.org/rtmapi/rtmapi/src/6d0996dc3d2683e976410ff759d23809a4ffcbfb/setup.py?at=default&fileviewer=file-view-default) it's not installed automatically. If I do not list the dependency here, the library does not work. I'll add a code comment.